.github/workflows/testing.yml

```diff
@@ -21,17 +28,22 @@ jobs:
       run:
         shell: bash
 
+    outputs:
+      output1: ${{ steps.step1.outputs.triggerdockerbuild }}
+
     container:
-      image: openmss/mss-test:latest
+      image: openmss/testing-${{ inputs.branch_name }}
 
     steps:
     - name: Trust My Directory
       run: git config --global --add safe.directory /__w/MSS/MSS
+
     - uses: actions/checkout@v2
-        
     - name: Reinstall dependencies if changed
+      id: step1
```

Comment:
There is not an enumeration of step1, step2 ..
Maybe rename this into check ?

Then this is like a question:

output1: ${{ steps.check.outputs.triggerdockerbuild }}

Reply:
the "steps:" is a keyword, we can't change, but the id: "lable" should be something we could give a meaningful name. In https://docs.github.com/en/actions/learn-github-actions/contexts a lable of "github_context_step" is used. 
A rename is not so important for me now,